sdcm/nemesis.py

```diff
@@ -2636,3 +2636,38 @@ def disrupt(self):
 #     @log_time_elapsed_and_status
 #     def disrupt(self):
 #         self.disable_disrupt_corrupt_then_scrub()
+
+
+class SlowHddNemesis(Nemesis):
+    def disrupt(self):
+        if isinstance(self.cluster, ScyllaAWSCluster):
+            raise UnsupportedNemesis("The nemesis works only on AWS cluster")
+        volume_name_char, size, volume_type = "f", 500, "st1"
```

Comment:
where this f is coming from ? should that be output of `create_and_attach_new_volume` ?

Reply:
To create new EC2 volume I need to send to API the volume name that can be `/dev/sd[a-z]`.
In my case I used in `f`.
Maybe I wrong and I can use any name I want?